src/main/resources/templates/fragments/topnav.html

```diff
@@ -1,7 +1,7 @@
 <div th:fragment="search-navbar">
     <nav class="navbar navbar-expand-lg navbar-light bg-light">
         <form class="form-inline my-2 my-lg-0">
-            <input class="form-control mr-sm-2" id = "pac-input" type="search" placeholder="Search" aria-label="Search">
+            <input class="form-control mr-sm-2" id = "pac-input" type="search" placeholder="Search"  aria-label="Search required">
             <button class="btn btn-outline-success my-2 my-sm-0" type="submit">Search</button>
```

Comment:
I put a required element in  here. Forcing the user to actually people info the search box.